pallets/liquidity-pools/src/message.rs

```diff
@@ -314,17 +293,19 @@ pub enum Message {
 		currency_payout: u128,
 		/// The amount of tranche tokens received for the investment made
 		tranche_tokens_payout: u128,
-		/// The remaining investment amount denominated in the `foreign` payment
-		/// currency. It reflects the sum of the unprocessed as well as the
-		/// processed but not yet collected amounts.
-		remaining_invest_amount: u128,
+		/// The fulfilled investment amount denominated in the `foreign` payment
+		/// currency. It reflects the amount of investments which were processed
+		/// independent of whether they were collected.
+		// TODO(@Luis): Apply delta instead of remaining to foreign investments
+		fulfilled_invest_amount: u128,
```

Comment:
In `FulfilledDepositRequest` message the fulfilled amount is not needed.

Anyway, I'm not sure if the solitary side has modified these fields yet. In that case, the error you're experiencing could be due to different format in these messages between us and solidity? (although sure you've already check that)